app/models/listing_unit.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: listing_units
+#
+#  id                  :integer          not null, primary key
+#  unit_type           :string(32)       not null
+#  translation_key     :string(64)
+#  transaction_type_id :integer
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+
+class ListingUnit < ActiveRecord::Base
+  attr_accessible(
+    :transaction_type_id,
+    :unit_type,
```

Comment:
Good point. `type` was my original idea for the name, but I decided to name it `unit_type`, because by default Rails does some special magic if the table contains column named `type`. It uses `type` for Single Table Inheritance.

At API level the `unit_type` is renamed to `type` so to the outside world the name is `type`, as it should.